js/rooms.js

```diff
@@ -45,7 +45,14 @@ $(document).ready(function() {
 				success: function(data) {
 					$('#app-navigation ul').html('');
 					data.forEach(function(element) {
-						$('#app-navigation ul').append('<li><a href="#'+escapeHTML(element['id'])+'">'+escapeHTML(element['name'])+' <span class="utils">' + escapeHTML(element['count']) + '</span></a></li>');
+						$('#app-navigation ul').append(
```

Comment:
there's an open PR https://github.com/nextcloud/spreed/pull/60 to replace this manual string concatenation with handlebars templating. This means the two PRs are conflicting 😟